### PR TITLE
Use platform-python on RHEL8 (#302)

### PR DESCRIPTION
**Pull request by swadeley**
/usr/libexec/platform-python bootstrap.py is the recommend way on RHEL8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,12 @@ When using the `--legacy-purge` option, a user account on the legacy environment
 * system group administrator for a system group that the system is a member of
 * granted permissions to the system explicitly via Users-> account-> 'Systems Administered by this User'
 
-# Usages:
+# Usage:
+
+On a RHEL8 host, call the bootstrap.py script as follows:
```
Comment (evgeni) [inline README.md]:
```suggestion
On an EL8 (RHEL8, CentOS8, etc) host, there is no `/usr/bin/python` or `/usr/bin/python3` by default. The `bootstrap.py` script can be used with the `platform-python` as follows:
```

File: bootstrap.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 Use `pydoc ./bootstrap.py` to get the documentation.
 Use `awk -F'# >' 'NF>1 {print $2}' ./bootstrap.py` to see the flow.
+Use `/usr/libexec/platform-python bootstrap.py` on RHEL8
```
Comment (evgeni) [inline bootstrap.py:8]: Is that important here?

**Review by evgeni — CHANGES_REQUESTED** [review]

File: bootstrap.py (hunk 2 above)
Comment (swadeley) [reply bootstrap.py:8]: no

Comment (swadeley) [reply bootstrap.py:8]: I think you can just merge and loose that or you prefer I force push to remove it.?

Comment (evgeni) [reply bootstrap.py:8]: Nah, I'm good.

**Review by evgeni — APPROVED** [review]

**Pull request merged**